ha/programmers-1/같은숫자는싫어.js

```diff
@@ -0,0 +1,3 @@
+function solution(arr) {
+  // 풀이
+}
```

Comment:
```suggestion
function solution(arr) {
  const test = 1;
}
```
- 여기서는 test 변수가 적절해보입니다.

Reply:
확인했습니다 or 변수 test보다는 apple이 좋을 것 같아요

Reply:
안녕하세요 연재님 🙇🏻‍♂️ 코드리뷰 진행했는데 한번 확인해주세요 🙆🏻‍♂️